CsvEnumerable/Classes/CsvEnumerable.cs

```diff
@@ -3,11 +3,11 @@
 using System.Collections.Generic;
 using System.IO;
 
-namespace Csv_Enumerable
+namespace CsvEnumerable
 {
     public class CsvEnumerable : IEnumerable<CsvRecord>, IEnumerator<CsvRecord>
     {
-        private bool disposed = false;
+        private bool disposed;
         int position = -1;
         List<CsvRecord> records;
```

Comment:
Consider using one style of declaring fields: either with 'private' or without it for all the fields